src/util/asset/assetUtils.ts

```diff
@@ -27,3 +28,35 @@ export async function generateOptIntoAssetTxns({
     );
   }
 }
+
+/**
+ * @param asset_1 - Asset 1 of the pool
+ * @param asset_2 - Asset 2 of the pool
+ * @returns Ordered (asset with bigger id becomes asset1) asset data of the pool in frontend friendly format
+ */
+export function prepareAssetPairData(
+  asset_1: Pick<TinymanAnalyticsApiAsset, "id" | "unit_name">,
+  asset_2: Pick<TinymanAnalyticsApiAsset, "id" | "unit_name">
+): {
+  asset1: {id: number; unitName: string};
+  asset2: {id: number; unitName: string};
+} {
+  const {unit_name: asset1UnitName} = asset_1;
+  const asset1ID = Number(asset_1.id);
+  const {unit_name: asset2UnitName} = asset_2;
+  const asset2ID = Number(asset_2.id);
+
+  // Make sure asset1 has greater ID
+  const assets =
+    asset1ID > asset2ID
+      ? {
+          asset1: {id: asset1ID, unitName: asset1UnitName},
+          asset2: {id: asset2ID, unitName: asset2UnitName}
+        }
+      : {
+          asset1: {id: asset2ID, unitName: asset2UnitName},
+          asset2: {id: asset1ID, unitName: asset1UnitName}
+        };
```

Comment:
Why don't we just use the object reference? This way we will keep all the data within the object
```suggestion
      ? {
          asset1: asset_1,
          asset2: asset_2
        }
      : {
          asset1: asset_2,
          asset2: asset_1
        };
```

Reply:
I think that's because `Pick<TinymanAnalyticsApiAsset, "id" | "unit_name">` shape (the input data shape) has asset id as string, but we want to convert them to number since we need number in js side.

```tsx
  const {unit_name: asset1UnitName} = asset_1;
  const asset1ID = Number(asset_1.id);
  const {unit_name: asset2UnitName} = asset_2;
  const asset2ID = Number(asset_2.id);
```